### ISSUE TITLE
Do we need Modernizr?

### ISSUE THREAD

**Issue by wrightmartin** [mysociety/pombola#1774]
Part of #1764

**Comment by wrightmartin** [comment]
There's a new version out, and it changes the way Modernizr works significantly, so it's worth looking at, especially if theres file size to be saved

https://modernizr.com/docs

**Comment by tmtmtmtm** [comment]
@wrightmartin — have you had a chance to look at the new version of Modernizr in any other context? Should we allocate some time to looking at it specifically for Pombola?

**Comment by wrightmartin** [comment]
@tmtmtmtm no, I haven't and yeah it would be good to have time allocated for it

**Comment by zarino** [comment]
This came up on Designer News today – http://featurejs.com – a Modernizr alternative that's only 1KB when gzipped (versus 4.5KB for the `modernizr_and_loader.js` that gets served on Mzalendo right now).

**Comment by dracos** [comment]
Pombola doesn't really use (or include) any of Modernizr's browser feature testing; Modernizr is included for its HTML5 shiv, `load` (a copy of yepnope), and `mq` media query testing. Looks like you could manually strip a few bits of Modernizr that aren't used (perhaps the minimiser does that already).

**Comment by zarino** [comment]
I just came across this same problem in FisMyStreet, while implementing https://github.com/mysociety/fixmystreet/issues/1350 – I wanted to check for flexbox support in the client, but FMS’s custom build of Modernizr 2.8.something doesn’t include that check. If I want to check for flexbox support, I’d have to make a new custom build, which basically means also upgrading to Modernizr 3 at the same time.

TLDR: Any research we do into the simplest way of getting conditional loading (yepnope), javascript media query testing, and javascript feature support testing, would be applicable to at least FMS as well.
